lib/ChunkGroup.js

```diff
@@ -290,16 +320,20 @@ class ChunkGroup {
 		return this._parents;
 	}
 
+	/**
+	 * @param {ChunkGroup} chunkGroup the parent group
+	 * @returns {boolean} returns true if this group has been removed from the parent
+	 */
 	removeParent(chunkGroup) {
 		if (this._parents.delete(chunkGroup)) {
-			chunkGroup.removeChunk(this);
+			chunkGroup.removeChild(this);
```

Comment:
I think this was a bug:

- `ChunkGroup#removeChunk` if for `Chunk` ❌
- `ChunkGroup#removeChild` is for `ChunkGroup`

Reply:
:+1: